infra/config-store/src/main/java/com/simprints/infra/config/store/remote/ConfigRemoteDataSourceImpl.kt

```diff
@@ -13,20 +13,19 @@ import javax.inject.Inject
 internal class ConfigRemoteDataSourceImpl(
     private val authStore: AuthStore,
     @DispatcherIO private val dispatcherIO: CoroutineDispatcher,
-    private val urlDownloader: (String) -> String
+    private val urlDownloader: (String) -> String,
 ) : ConfigRemoteDataSource {
 
     @Inject
     constructor(
         authStore: AuthStore,
-        @DispatcherIO dispatcherIO: CoroutineDispatcher
+        @DispatcherIO dispatcherIO: CoroutineDispatcher,
     ) : this(authStore, dispatcherIO, { url -> URL(url).readText() })
 
-    override suspend fun getConfiguration(projectId: String): ProjectConfiguration =
-        getApiClient().executeCall { it.getConfiguration(projectId) }.toDomain()
-
-    override suspend fun getProject(projectId: String): Project =
-        getApiClient().executeCall { it.getProject(projectId) }.toDomain()
+    override suspend fun getProject(projectId: String): Pair<Project, ProjectConfiguration> =
+        getApiClient()
```

Comment:
Adding a wrapping class here makes even more sense since we've removed the `getConfiguration` function. 

Defining `fun getProject(projectId: String): ProjectWithConfig` is way more appropriate in my opinion than `fun getProject(projectId: String): Pair<Project, ProjectConfiguration>`